dht/token.cc

```diff
@@ -288,4 +288,14 @@ compaction_group_of(unsigned most_significant_bits, const token& t) {
     __builtin_unreachable();
 }
 
+token last_token_of_compaction_group(unsigned most_significant_bits, size_t group) {
```

Comment:
Should it be used in `dht::split_token_range_msb`?

Reply:
the latter is called by `table::make_compaction_groups`

Reply:
Yes, split_token_range_msb() can be implemented with this.